content/episodes/2019-03-04-S04E07-what-if-women-built-the-internet.md

```diff
@@ -0,0 +1,234 @@
+
+  **Manoush Z.:** What did you do?
+
+  **Coraline AE:** I can't get into the specifics of what I did. I did everything in my power to keep myself safe. I have to maintain very high online and personal security.
+
+  **Manoush Z.:** Why do people do you think feel so threatened by this idea of welcoming all and being very specific about it and having a code of conduct?
+
+  **Coraline AE:** Because it challenges the status quo and there's a portion of the population in tech who unduly benefit from the status quo because of an accident of birth. Those people who benefit from privilege, unexamined privilege balk at the idea that they have an advantage over other people given to them, that they're playing life on an easier mode than other people are playing.
+
+  Anything that threatens their position within that community or their feeling of accomplishment in having made it on their own, they find very threatening and they tend to lash out at that very idea.
+
+  **Manoush Z.:** How many people now have signed your code? And what are companies saying about adopting it?
+
+  **Coraline AE:** I am very happy to report that we topped the 100,000 mark. So over 100,000 adoptions of the Contributor Covenant, largely from smaller or community organized open source projects but this year, saw Apple, Google, Intel, and Microsoft adopting Contributor Covenant for all of their open source projects and most recently, the Linux kernel adopted Contributor Covenant for their work as well.
+
+  **Manoush Z.:** Okay, that's the back end. Very encouraging. What about the front end? What difference does it make to those of us who are not open source coders, we just use the web? Do we see a difference?
+
+  **Coraline AE:** If you're just solving problems for the 25 year old white male with high income, high mobility, fast internet access, you're leaving a lot of people behind and it's really hard for you to sort of think outside of your own lived experience.
+
+  **Manoush Z.:** What would the internet look like if it was built by women?
+
+  **Coraline AE:** I think - I'd like to think that women have a better understanding of consent than a lot of men do and I think if women built the internet, consent would've been baked into it. We wouldn't see data breaches where private information was released because we wouldn't be collecting information without people's explicit consent.
+
+  We would see a reduction in harassment because engaging with someone on social media would be more consent oriented as opposed to anyone can respond to anything that I say at any point in time.
+
+  **Manoush Z.:** Coraline Ada Ehmke has been an open source programmer for over 20 years and created the Contributor Covenant. If you wanna check it out, you will find a link in our show notes. You can also learn about Mozilla's own community participation guidelines.
```

Comment:
There aren't any links to the Contributor Covenant or the Mozilla CPG in the show notes. Did we miss a section?

https://www.contributor-covenant.org/
https://www.mozilla.org/about/governance/policies/participation/

Reply:
I have updated show notes from Michaela with these links included.